modules/main-rules/pages/assault-moves-and-charges.adoc

```diff
@@ -27,6 +27,12 @@ This does not necessarily mean the *closest* enemy unit, or even one that is in
 TIP: You do not have to move units more than 5 cm if you do not want to.
 You might only want to move units into range for a firefight or to grab an objective, for example.
 
+[NOTE]
+.Units that can't move
+====
+The requirement to move at least 5 cm or into base contact does not override any rules that prevent the unit from moving, for example if the detachment fails a Leadership test to move, or the unit fails the first part of a Dangerous terrain test.
+====
+
```

Comment:
I think that this part of the topic is getting quite 'noisy' with note blocks. Given that this is really only a clarification that a player will probably only ever need to read once, I think that we could relegate it to a footnote. Possibly also change the relevant line to something like `You *must* move your unit at least 5 cm if you are free to do so -- ...`
I might try that and see how it looks.

Reply:
This conversation is now outdated by change in which I've implemented the footnote approach. So I'll resolve.